Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -633,6 +640,9 @@ namespace AZ
                 {
                     m_meshFeatureProcessor->SetVisible(m_meshHandle, visible);
                 }
+#if defined(CARBONATED)
+                m_configuration.m_visibility = visible;
+#endif
                 m_isVisible = visible;
```

Comment:
Do we need this extra flag now that the configuration seems to track it instead?

Reply:
Yeah that came up when I realized how they update the local settings in editor, you can see the same pattern for the sort key

Reply:
I personally would just use the config, but i figured if this was stylistically matching